tests/responses/profiles_list.json

```diff
@@ -0,0 +1,67 @@
+{
+  "_embedded": {
+    "profiles": [
+      {
+        "resource": "profiles",
+        "id": "pfl_v9hTwCvYqw",
+        "mode": "live",
+        "name": "My website name",
+        "website": "https://www.mywebsite.com",
+        "email": "info@mywebsite.com",
+        "phone": "+31208202070",
+        "categoryCode": 5399,
+        "status": "verified",
+        "review": {
+          "status": "pending"
+        },
+        "createdAt": "2018-03-20T09:28:37+00:00",
+        "_links": {
+          "self": {
+            "href": "https://api.mollie.com/v2/profiles/pfl_v9hTwCvYqw",
+            "type": "application/hal+json"
+          },
+          "chargebacks": {
+            "href": "https://api.mollie.com/v2/chargebacks?profileId=pfl_v9hTwCvYqw",
```

Comment:
As these links represent subsets of the the regular endpoint, specific to the profile, it may be nice to have these available as properties too, i.e. `profile.chargebacks`. Same for the methods, payments, refunds and checkoutPreviewUrl.

Reply:
Done